src/pages/SchedulePublic/SchedulePublicPage.css

```
@@ -537,4 +660,153 @@
+  background: rgba(239,68,68,0.08);
+  border-color: #dc2626;
+  color: #dc2626;
+}
+
+/* Item đang quan tâm — border highlight nhẹ */
+.sp-item--followed {
+  box-shadow: 0 0 0 1.5px rgba(85, 197, 241,0.2), var(--shadow-sm);
+}
+
+/* Stat quan tâm */
+.sp-stat-num--followed {
```

Stylelint (17.5.0)

[error] Unclosed block (CssSyntaxError)

(CssSyntaxError)